### PR TITLE
linalg: add support for clamp_min and clamp_max ops

### PR DESCRIPTION
**Pull request by ashay**
This patch adds two new operations (`AtenClampMinOp` and
`AtenClampMaxOp`) to the list of generated torch operations, along with
a corresponding translation to linalg-on-tensors.

### CODE REVIEW

File: lib/Conversion/TorchToLinalg/Uncategorized.cpp [hunk 1]
```diff
@@ -836,15 +867,15 @@ class ConvertElementwiseOp : public ConversionPattern {
...
-             AtenNeScalarOp, AtenNegOp, AtenMaskedFillScalarOp>(op))
+             AtenClampOp, AtenClampMinOp, AtenClampMaxOp, AtenRsubScalarOp,
+             AtenMulScalarOp, AtenLogOp, AtenErfOp, AtenSqrtOp, AtenFloorOp,
+             AtenPowTensorScalarOp, AtenLog2Op, AtenRsqrtOp, AtenDivScalarOp,
+             AtenAbsOp, AtenReciprocalOp, AtenBitwiseAndTensorOp,
+             AtenGtScalarOp, AtenGeScalarOp, AtenEqScalarOp, AtenLtScalarOp,
+             AtenLeScalarOp, AtenWhereSelfOp, AtenCeilOp, AtenGtTensorOp,
+             AtenEqTensorOp, AtenLtTensorOp, AtenSubScalarOp, AtenAddScalarOp,
+             AtenThresholdOp, AtenThresholdBackwardOp, AtenCloneOp, AtenSinOp,
+             AtenCosOp, AtenNeScalarOp, AtenNegOp, AtenMaskedFillScalarOp>(op))
```
Comment (ashay) [inline lib/Conversion/TorchToLinalg/Uncategorized.cpp:878]: This simply adds `AtenClampMaxOp` and `AtenClampMinOp` close to `AtenClampOp` to keep them all together.

File: lib/Dialect/Torch/Transforms/RefineTypes.cpp [hunk 2]
```diff
@@ -624,23 +624,23 @@ ChangeResult TypeAnalyzer::visitOperation(
           AtenReluOp, AtenGeluOp, AtenCeilOp, AtenGeluBackwardOp,
           AtenBitwiseNotOp, AtenToPrimDeviceOp, AtenCpuOp, AtenContiguousOp,
           AtenFill_ScalarOp, AtenDetachOp, AtenMaskedFill_ScalarOp, AtenCopy_Op,
-          AtenCumsumOp, AtenLayerNormOp, AtenClampOp, AtenNegOp, AtenFloorOp,
```
Comment (ashay) [inline lib/Dialect/Torch/Transforms/RefineTypes.cpp:627]: Same here.  This simply adds `AtenClampMaxOp` and `AtenClampMinOp` close to `AtenClampOp` to keep them all together.

**Review by ramiro050 — APPROVED** [review]
Thanks for this! LGTM!

**Review by ramiro050 — CHANGES_REQUESTED** [review]
I forgot, can you add your ops to the `illegal` list:

https://github.com/llvm/torch-mlir/blob/5705e52c631246c83e327c3e4e8b5b4d52ba6590/lib/Conversion/TorchToLinalg/Uncategorized.cpp#L1605

**Comment by ashay** [comment]
Closing this in favor of PR #899.

**Pull request closed without merging**